### ISSUE TITLE
Custom icon not loaded in Launcher only

### ISSUE THREAD

**Issue by Carlisle96** [pop-os/launcher#93]
I have the following setup:

PopOS 21.10
My own single icon created in ~/.icons
![image](https://user-images.githubusercontent.com/33028362/159156585-e5c06c72-429a-4580-ba8f-2622a5049f47.png)

The icon gets loaded correctly in the ~/.icons folder because it is shown correctly in the applications overview:
![image](https://user-images.githubusercontent.com/33028362/159156702-8f068fb8-3f77-4884-a101-e77525e45ec2.png)

But in the launcher the placeholder icon is shown:
![image](https://user-images.githubusercontent.com/33028362/159156722-53e5681f-d97c-483a-9b83-7568a2b9f68a.png)

What am I doing wrong ? 
Was my icon installed correctly ?
How can I make the icon load correctly on the launcher too ?

**Comment by jacobgkau** [comment]
> Was my icon installed correctly ?

The icon should be specified in the `.desktop` file for that Datev application. Can you post the contents of the `.desktop` file?

**Comment by mmstick** [comment]
How did you install this?

**Comment by Carlisle96** [comment]
> How did you install this?
I am running Pop OS 21.10, hence it was installed by default.

**Comment by mmstick** [comment]
@Carlisle96 This isn't part of a default Pop install.

**Comment by Carlisle96** [comment]
I am talking about this Launcher 
![image](https://user-images.githubusercontent.com/33028362/162589244-57ff6d39-5107-4e33-994f-30b1a0874498.png)
I thought this got added to Pop OS in 21,10 and I did not install any extensions for it.

**Comment by mmstick** [comment]
I'm talking about this application you're having issues with

**Comment by Carlisle96** [comment]
It happens with every application I select a selft created icon for in the *.desktop file. Independent of the application. The one i showed is just an example.

**Comment by halfmexican** [comment]
Yeah, it appears the launcher doesn't always play nice with certain icon theme. I'm using papirus
![image](https://user-images.githubusercontent.com/103920890/226033772-ac604398-4bf9-4423-b873-c17acd457a0c.png)

**Comment by halfmexican** [comment]
It appears to be an issue with how the desktop entry is set up

**Comment by liias** [comment]
> > The icon should be specified in the `.desktop` file for that Datev application. Can you post the contents of the `.desktop` file?
> 
> I don't think the .desktop file is configured wrongly, because it works in the application menu. Here are the contents: ![image](https://user-images.githubusercontent.com/33028362/159312753-b6f1a6d5-cb19-4010-8112-36d74593c1f7.png)

`Icon` in `.desktop` file shouldn't have file extension `.svg` 
```
Icon=datev2
```

unless you specify it as an absolute path:
```
Icon=/home/thyriaen/.icons/datev2.svg
```
